java.lang.RuntimeException: Nonzero exit code: 128
	at scala.sys.package$.error(package.scala:27)
	at sbt.Streamed$.sbt$Streamed$$next$1(ProcessImpl.scala:429)
	at sbt.Streamed$$anonfun$apply$10.apply(ProcessImpl.scala:432)
	at sbt.Streamed$$anonfun$apply$10.apply(ProcessImpl.scala:432)
	at sbt.AbstractProcessBuilder.lines(ProcessImpl.scala:151)
	at sbt.AbstractProcessBuilder.lines(ProcessImpl.scala:141)
	at $e14ef6aeb5a204d222f8$$anonfun$$sbtdef$1.apply(/Users/abhishek.srivastava/MyProjects/sbt-test/build.sbt:15)
	at $e14ef6aeb5a204d222f8$$anonfun$$sbtdef$1.apply(/Users/abhishek.srivastava/MyProjects/sbt-test/build.sbt:15)
	at sbt.std.Transform$$anon$3$$anonfun$apply$2.apply(System.scala:44)
	at sbt.std.Transform$$anon$3$$anonfun$apply$2.apply(System.scala:44)
	at sbt.std.Transform$$anon$4.work(System.scala:63)
	at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:226)
	at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:226)
	at sbt.ErrorHandling$.wideConvert(ErrorHandling.scala:17)
	at sbt.Execute.work(Execute.scala:235)
	at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:226)
	at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:226)
	at sbt.ConcurrentRestrictions$$anon$4$$anonfun$1.apply(ConcurrentRestrictions.scala:159)
	at sbt.CompletionService$$anon$2.call(CompletionService.scala:28)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:471)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
[0m[[31merror[0m] [0m(*:[31mgitHeadCommitSha[0m) Nonzero exit code: 128[0m
